lib/tapioca.rb

```diff
@@ -15,6 +22,25 @@ def self.silence_warnings(&blk)
   end
 
   class Error < StandardError; end
+
+  SORBET_PATH = T.let("sorbet", String)
+  SORBET_CONFIG = T.let("#{SORBET_PATH}/config", String)
+  TAPIOCA_PATH = T.let("#{SORBET_PATH}/tapioca", String)
+  TAPIOCA_CONFIG = T.let("#{TAPIOCA_PATH}/config.yml", String)
+
+  DEFAULT_COMMAND = T.let("bin/tapioca", String)
+  DEFAULT_POSTREQUIRE = T.let("#{TAPIOCA_PATH}/require.rb", String)
+  DEFAULT_RBIDIR = T.let("#{SORBET_PATH}/rbi", String)
+  DEFAULT_DSLDIR = T.let("#{DEFAULT_RBIDIR}/dsl", String)
+  DEFAULT_GEMDIR = T.let("#{DEFAULT_RBIDIR}/gems", String)
+  DEFAULT_SHIMDIR = T.let("#{DEFAULT_RBIDIR}/shims", String)
+  DEFAULT_TODOSPATH = T.let("#{DEFAULT_RBIDIR}/todo.rbi", String)
```

Comment:
Again, `PATH` vs `DIR`.

```suggestion
  DEFAULT_TODOS_PATH = T.let("#{DEFAULT_RBI_DIR}/todo.rbi", String)
```

Should we use `PATH` (or maybe `FILE`?) for a file and `DIR` for a directory?